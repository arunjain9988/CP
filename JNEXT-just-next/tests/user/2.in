1
10
1 15 2 3 5 6 89 56 23 23