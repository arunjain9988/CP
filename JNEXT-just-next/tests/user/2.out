115235236235689
